src/js/edu-benefits/components/SubmitMessage.jsx

```diff
@@ -8,6 +8,13 @@ export default class SubmitMessage extends React.Component {
     this.state = { isExpanded: false };
     this.handleClick = this.handleClick.bind(this);
   }
+  componentDidMount() {
+    const title = document.querySelector('.edu-page-title');
```

Comment:
Wondering if we want to extract out this functionality into something that can be shared across components?